qa/build.gradle

```diff
@@ -5,13 +5,13 @@ subprojects { Project subproj ->
   subproj.tasks.withType(RestIntegTestTask) {
     if (subproj.extensions.findByName("${it.name}Cluster")) {
       subproj.extensions.configure("${it.name}Cluster") { cluster ->
-        cluster.distribution = System.getProperty('tests.distribution', 'oss')
+        cluster.distribution = System.getProperty('tests.distribution', 'default')
```

Comment:
Is this intentional? If we are to do this, it should be a separate change. Where would we be running these tests with the oss distribution?

Reply:
Indeed. As I understand it we run everything against OSS and then we have a specific build to use the default distribution. Have we inverted this?

Reply:
The current jobs no longer assume a default, they pass this in explicitly. 
The existing default is not consistent, we have the default  distro for docs and oss for the qa directory.  I find this to be confusing and hard to reason about coverage when looking at the jobs, so this change makes it consistent that we always default to the default distribution for the sole reason that  it's a super set.
You'll see that some of the existing jobs don't have the correct coverage due to this confusion. 
I think it's ok to do this as part of this change, because the new jobs do provide the correct coverage and the old ones are meant to go away very soon after we merge this.